tumblr-downloader.go

```diff
@@ -150,67 +154,110 @@ func tumblrGetLikes(httpClient *http.Client, blogIdentifier string, timestamp in
 	return parser
 }
 
-func downloadURL(URL string) {
-	resp, err := http.Get(URL)
-	checkError(err)
+func downloadURL(URL string, SubDir string, AppendExtension string) {
+	fmt.Printf("Downloading %s ... ",URL)
```

Comment:
A agree. Logging mechanism needs to be reworked. Current is overly intrusive and interactive.
We need a counter that allows you to see how far the download is progressing.